### ISSUE TITLE
Media: consider adding a Media top-level navigation link in sidebar

### ISSUE THREAD

**Issue by lancewillett** [Automattic/wp-calypso#8346]
Consider adding a direct link to the Media Library view from Menu (sidebar) — right now it's only accessible from the Editor view — and wp-admin has the direct link.

@folletto says

> This is an interesting one. We discussed a bit, and afaik there was no consensus if we wanted to have it as a separate item on the sidebar. Maybe as a sub-item from Posts? Not sure, we still have to explore the solutions here.

#calypso-gaps

**Comment by folletto** [comment]
We need to be very very careful in adding *anything* to the side navigation. I know it seems obvious, and I know it's "already like that" in WordPress, but we are already at the edge of a list of items that is too long, getting us back in the situation where it's an overwhelming experience.

We surely have to address this, but I'd rather explore first alternatives that aren't a top-level navigation item.

So I'd rephrase this ticket if possible as "Explore alternatives", not as a direct suggestion, if that's ok.

**Comment by mtias** [comment]
See also #8381.

**Comment by gwwar** [comment]
@lancewillett this is currently on Stark's roadmap. Related issues will likely be organized in https://github.com/Automattic/wp-calypso/projects/5 

Should we close this as dup, or use it as a tracking issue for your milestone?

**Comment by lancewillett** [comment]
Thanks, Kerry. Let's close as a dupe.
